plat/marvell/a8k/common/a8k_common.mk

```diff
@@ -13,6 +13,9 @@ PLAT_COMMON_BASE	:= $(PLAT_FAMILY_BASE)/common
 MARVELL_DRV_BASE	:= drivers/marvell
 MARVELL_COMMON_BASE	:= plat/marvell/common
 
+SVC_TEST		:= 0
```

Comment:
You should prefix this option by MARVELL.

Reply:
Done